### PR TITLE
Fix #382 , Implement deleting done checklist items

### LINKED ISSUE #382: [feature request] add button to delete done list items
Hey thanks for your great work. I use many of your simple apps and I really like them.
I would propose a feature for simple notes:
add a button or option to delete all done options of a checklist.
This would make it easier for shopping lists to keep an overview.

Keep up your good work :)

### PR DESCRIPTION
**Pull request by tanvir-ahmod**
Done items of checklist can be deleted in one click from the menu

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt [hunk 1]
```diff
@@ -241,7 +243,8 @@ class MainActivity : SimpleActivity() {
         }
     }
 
-    private fun isCurrentItemChecklist() = mAdapter?.isChecklistFragment(view_pager.currentItem) ?: false
+    private fun isCurrentItemChecklist() = mAdapter?.isChecklistFragment(view_pager.currentItem)
+            ?: false
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt:247]: undo these formatting changes please, it is hard to see what really changed

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -101,6 +101,7 @@
         <b>Reddit:</b>
         https://www.reddit.com/r/SimpleMobileTools
     </string>
+    <string name="remove_done_items">Remove done items</string>
```
Comment (tibbi) [inline app/src/main/res/values-ar/strings.xml:104]: move this new string to Checklists section please, not to the bottom

File: app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt (hunk 1 above)
Comment (tanvir-ahmod) [reply app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt]: Please check now

File: app/src/main/res/values-ar/strings.xml (hunk 2 above)
Comment (tanvir-ahmod) [reply app/src/main/res/values-ar/strings.xml]: Done

**Comment by tibbi** [comment]
okay, the code itself looks good, I will test it a bit once Im working on the app, might take a couple days/weeks. It wont be updated earlier anyway. Thanks

**Comment by tibbi** [comment]
alright, Ill tweak a few things here and there, but looks good in general :) Thanks

**Pull request merged**